sdcm/nemesis.py

```diff
@@ -2053,6 +2053,11 @@ def get_ks_with_few_tables(keyspace_table):
         self.log.debug(result)
 
     def disrupt_show_toppartitions(self):
+        # choose old or new syntax for toppartitions command
+        # old syntax allow get info about only one ks:cf
+        # new syntax allow get info about one, several or all
+        use_new_syntax = random.choice([True, False])
+
         def _parse_toppartitions_output(output):
```

Comment:
great job, but let's move out toppartitions related code into separate file, convert it into two classes that handle new style and old style.

Reply:
@dkropachev COuld you review?